setup.py

```diff
@@ -25,7 +25,7 @@
       zip_safe=False,
       setup_requires=['pytest-runner'],
       tests_require=['pytest', 'pytest-mock', 'pytest-flask', 'pytest-aiohttp'],
-      install_requires=['apispec', 'schematics', 'more_itertools'],
+      install_requires=['apispec', 'schematics', 'more_itertools', 'numpy', 'Werkzeug', 'jinja2'],
```

Comment:
I really dislike having requirements specified both in req.txt and here...

Reply:
Yeah, the packaging situation in Python sucks pretty hard now. Maybe `pipenv` will save us one day.